lib/tasks/identity_types.rake

```diff
@@ -0,0 +1,12 @@
+namespace :goodcity do
+  desc 'Initialize the different identity types'
+  task create_identity_types: :environment do
+    id_types = YAML.load_file("#{Rails.root}/db/identity_types.yml")
+    id_types.each do |index, record|
```

Comment:
If you don't need to user `index` argument than you can use `each_value` instead of each